src/edge/importer.py

```diff
@@ -142,10 +169,13 @@ def _annotate_feature(self, fragment, first_base1, last_base1, name, type, stran
                 (name, first_base1, last_base1))
 
         annotation_start = self.__fclocs[first_base1]
-        if last_base1 != len(self.__sequence):
+        if last_base1 < len(self.__sequence):
             annotation_end = self.__fclocs[last_base1 + 1]
-        else:
+        elif last_base1 == len(self.__sequence):
             annotation_end = self.__fclocs[1]
+        else:
+            # bad annotation, beyond end of sequence
```

Comment:
Do we need to throw any warnings here or does it not matter at all?

Reply:
I don't know how to throw a warning to the user.

Reply:
It's fine to keep it this way for now. I think in the future if we have the bandwidth, we might want to introduce logger so we can have different levels of logging where the other applications can catch. for example, logger.info, logger.warning, etc.